src/egl/drivers/dri2/platform_android.c

```diff
@@ -476,6 +476,18 @@ droid_destroy_surface(_EGLDriver *drv, _EGLDisplay *disp, _EGLSurface *surf)
    return EGL_TRUE;
 }
 
+static EGLBoolean droid_swap_interval(_EGLDriver *drv, _EGLDisplay *dpy,
+        _EGLSurface *surf, EGLint interval) {
+
+   struct dri2_egl_surface *dri2_surf = dri2_egl_surface(surf);
```

Comment:
@Zhongmin Wu can you rebase this on top of master and send a fresh PR ?

Reply:
@zhongmin Are you sending it to mesa mailing list for review ? @tpalli ping

Reply:
Is this OK ? If OK, I will prepare a patch to mesa mailing list.